dist_tables/ddl.rst

```diff
@@ -61,6 +61,45 @@ Each created shard is assigned a unique shard id and all its replicas have the s
 
 You are now ready to insert data into the distributed table and run queries on it. You can also learn more about the UDF used in this section in the :ref:`user_defined_functions` of our documentation.
 
+.. _colocation_groups:
+
+Co-Location Groups
+------------------
+
+By default two tables are co-located when they are distributed by columns of the same type using the same distribution method and over the same number of shards. Another way to say this is that the tables are in the same *co-location group.*
```

Comment:
Citus defaults to hash distribution and the same number of shards. Is it worth noting ".. using the same distribution method and over the same number of shards" at the beginning?

What if we pushed this part to later in the section (exceptions / when you change the defaults), etc?